diff --git a/sqlite3.c b/sqlite3.c
index 55dc686..bf3e9c3 100644
--- a/sqlite3.c
+++ b/sqlite3.c
@@ -37485,13 +37485,7 @@ static int unixDeviceCharacteristics(sqlite3_file *id){
 ** Instead, it should be called via macro osGetpagesize().
 */
 static int unixGetpagesize(void){
-#if OS_VXWORKS
-  return 1024;
-#elif defined(_BSD_SOURCE)
   return getpagesize();
-#else
-  return (int)sysconf(_SC_PAGESIZE);
-#endif
 }
 
 #endif /* !defined(SQLITE_OMIT_WAL) || SQLITE_MAX_MMAP_SIZE>0 */
